pages/1.10/security/ent/perms-reference/index.md

```diff
@@ -158,8 +158,8 @@ Marathon applications using secrets see [Configuring services and pods to use se
 ## <a name="superuser"></a>Superuser Permissions
 
 Similar to the Windows `Administrator` or Linux `root` accounts, DC/OS has the
-concept of the `superuser`. A user with permission to perform any action on the `dcos:superuser` resource has complete, unrestricted access to any operation
-throughout DC/OS. This is extremely powerful and this permission should be
+concept of the `superuser`. A user with at least one permission out of `create`, `read`, `update`, `delete` or `full` on the `dcos:superuser` resource has complete, unrestricted access to any operation
+throughout DC/OS. This is counter-intuitive and extremely powerful. Therefore this permission should be
```

Comment:
Why is this counter-intuitive?

Reply:
I think we should leave out the "counter-intuitive". It just raises the question why have we made it counter-intuitive?